src/main/scala/gitbucket/core/plugin/IssueHook.scala

```diff
@@ -13,6 +13,10 @@ trait IssueHook {
     implicit session: Session,
     context: Context
   ): Unit = ()
+  def updatedComment(commentId: Int, content: String, issue: Issue, repository: RepositoryInfo)(
```

Comment:
Hi, @onukura 

The work you did looks really good. I'm just wondering what the purpose of this function is. I'm totally new to this project so maybe you can explain this to me.

This method doesn't seem to have an overrided implementation anywhere, what exactly does it do?

Reply:
@jyoo980 Hi! actually i am new too ;) So I probably can not answer well for your question. 
This is hook point which will be called when comment update happen. This kind of Hooks are often used by Plugin.
for example gitbucket-notification-plugin use IssueHook

https://github.com/gitbucket/gitbucket-notifications-plugin/blob/b51f194d2b7df2639471a3cc7a91441cb627607c/src/main/scala/gitbucket/notifications/service/NotificationsHook.scala#L57-L79

So I added Hook point for updated comment event as there was not.

@takezoe Is it ok to add `updatedComment` hook point to IssueHook?

Reply:
Ah so it's actually being overriden by a separate project. I see, thanks so much for the clarification!

Reply:
Adding this hook method is beneficial for plugins. Thanks!